src/components/table/table.html

```diff
@@ -0,0 +1,39 @@
+<table class="m-table" :class="[ 'm-table-' + skin]" cellspacing="0">
+    <slot name="header">
+        <thead>
+            <tr>
+                <slot v-for="column in columns" :name="'header.' + column.dataProp">
+                    <th :style="columnWidth(column)">
+                        {{ column.title }}
+                    </th>
+                </slot>
+            </tr>
+        </thead>
+    </slot>
+
+    <slot name="body">
+        <tbody>
+            <tr v-if="isEmpty">
+                <slot name="empty">
+                    <td class="m-table--empty" :colspan="columns.length">
+                        <m-icon name="m-svg__data" size="55"></m-icon>
+                        <p>{{ i18nEmptyTable }}</p>
+                    </td>
+                </slot>
+            </tr>
+            <tr v-else v-for="row in rows">
+                <slot v-for="column in columns" :name="'body.' + column.dataProp" :data="row">
```

Comment:
toujours ajouter un :key au v-for , voir-> https://vuejs.org/v2/style-guide/#Keyed-v-for-essential